apps/web/src/pages/quick-start/components/demo-flow/nodes/NodeStep.tsx

```diff
@@ -7,6 +7,7 @@ import { colors, shadows, Text } from '../../../../../design-system';
 import { guidePreview, GuideTitleEnum, IBeat, IGuide } from './consts';
 import { useSegment } from '../../../../../components/providers/SegmentProvider';
 import { When } from '../../../../../components/utils/When';
+import { AddNodeContainer } from '../../../../templates/workflow/workflow/node-types/AddNode';
```

Comment:
import cross module is a bad thing, if we want something shared we should define it in the components folder

Reply:
Good point I will extract the common component.